anyrun.json

```diff
@@ -180,9 +180,10 @@
         {
             "action": "detonate file",
             "description": "Run the file in the sandbox and returns the ID of created task",
-            "type": "generic",
+            "verbose": "For the 'opt_timeout' parameter Size range: 10-660. For the 'obj_ext_cmd' Size range: 2-256.",
```

Comment:
Should these params restriction be mentioned in the description of those 2 params as well? 
Users will be able to see the description right next to their inputs.

Reply:
Fixed